e2e-tests/src/NPG/wallet.npg.onboarding.integration.test.ts

```diff
@@ -0,0 +1,43 @@
+import { fillCardDataForm } from './helper';
+
+describe('wallet payment npg activation tests', () => {
+  /**
+   * Test input and configuration
+   */
+  const NPG_WALLET_URL = String(process.env.NPG_WALLET_URL);
+  const VALID_CARD_DATA = {
+    number: String(process.env.CARD_NUMBER_XPAY),
+    expirationDate: String(process.env.CARD_EXPIRATION_DATE_XPAY),
+    ccv: String(process.env.CARD_CCV_XPAY),
+    holderName: String(process.env.CARD_HOLDER_NAME_XPAY),
+  };
+
+  const sessionToken = 'exampleToken';
```

Comment:
A call to the BE should be made to simulate the creation of a wallet so that security tokens and walletIDs are valid in the various environments

Reply:
Done. The only problem is that at the moment the onboarding is not working in the published environments, to test the newest change I'had to make a string replace on the redirectUrl: changing the host with localhost. More details in this pr description.